ion/systemd/ion-nm-agent.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
```

Comment:
Does relaxing this work consistently to have the local agent register with the manager? This was put in place because the agent attempts exactly one time at startup to register, so it must happen after the manager is up and ready.